phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexUpgradeTool.java

```diff
@@ -707,12 +716,12 @@ private int startIndexRebuilds(HashMap<String, IndexInfo> indexInfos,
             list.add("-tenant");
             list.add(tenantId);
         }
-        if (syncRebuild) {
-            list.add("-runfg");
-        }
-        if(!Strings.isNullOrEmpty(verify)) {
-            list.add("-v");
-            list.add(verify);
+
+        if(!Strings.isNullOrEmpty(indexToolOpts)) {
+            String[] options = indexToolOpts.split("\\s+");
```

Comment:
Is this still a tab and not space? diff looks different from `238 to 242`. Please consider using phoenix code template if not already: https://phoenix.apache.org/develop.html